.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
essas configurações já não podem ser inseridas no prettier?

Reply:
https://prettier.io/docs/en/options.html#tabs

> Setting indent_style in an [.editorconfig file](https://editorconfig.org/) will configure Prettier’s tab usage, unless overridden.

(Tabs will be used for indentation but Prettier uses spaces to align things, such as in ternaries.)